InterfacePython/testAddons.py

```diff
@@ -136,8 +136,43 @@ def checkUnstructuredL2(self):
 
         self.assertLess(np.max(np.abs(model - surrogate[:,0])), 5.E-3, "Constructed from unstructured too inaccurate.")
 
+    def checkExoticQuadrature(self):
+        # Initialize some useful constants and utility functions.
+        integrand = lambda x : np.exp(-np.linalg.norm(x) ** 2)
+        sinc1s0 = lambda x : np.sinc(x / np.pi)
+        sinc10s1 = lambda x : np.sinc(10 * (x - 1) / np.pi)
+        max_level = 25
+        nref = 101
+        def create_surrogate(lambda_fn):
+            grid = Tasmanian.makeGlobalGrid(1, 1, nref, "level", "gauss-legendre")
+            grid.loadNeededValues(np.array([lambda_fn(x) for x in grid.getNeededPoints()]))
```

Comment:
You can also write this without the for operation, see the comment below. You have to be more careful about the dimensions here.

Reply:
Done. I changed this to:
```
grid.loadNeededValues(np.apply_along_axis(lambda_fn, 1, grid.getNeededPoints()))
```
to be in line with the above comment.